tests/docker/ducktape-deps/apache-flink

```diff
@@ -0,0 +1,20 @@
+set -e
+
+# update your system and install Java
+sudo apt -y update && sudo apt -y install openjdk-11-jdk
```

Comment:
I believe `base` already contains java (we run a bunch of other java based clients) so we can get rid of this step and save some time